HealthcareBase/Service/ScheduleService/PatientRecommendationService/PrioritiseDoctorStrategy.cs

```diff
@@ -13,13 +13,13 @@ namespace HealthcareBase.Service.ScheduleService.PatientRecommendationService
 {
     public class PrioritiseDoctorStrategy : RecommendationStrategy
     {
-        public ProcedureType PatientDefault { get; set; }
+        public ProcedureDetails PatientDefault { get; set; }
 
         public ProcedurePreferenceDTO TransformRequest(RecommendationRequestDTO request)
         {
             var preference = new ProcedurePreferenceDTO
             {
-                Type = PatientDefault,
+                Details = PatientDefault,
                 Patient = request.Patient,
                 Preference = new ProcedureSchedulingPreference
                 {
```

Comment:
Try to refactor this method because same lines are repeating in 3 different strategies.

Reply:
This line is from an unused file, changes happened because i was refactoring some class which was used by this one. It is probably going to get deleted.